errors.js

```diff
@@ -1,15 +1,17 @@
 exports.handleCustomErrors = (err, req, res, next) => {
-    if (err.status && err.msg) {
-      // Respond with the error status and a message property
-      res.status(err.status).send({ message: err.msg });
-    } else {
-      next(err);
-    }
+  if (err.status && err.message) {
+    console.log(err.msg, "mmmmmm");
```

Comment:
Generally, just for future reference, we want to make sure that any stray console.logs are deleted before we put our code up for code review. Reasons being is just that we wouldn't expect to see console.logs in production, and putting code up for a code review is essentially saying "This code can be merged into main/production right now" :)